src/AnnouncementBubble/QuotedAnnouncementBubble.tsx

```diff
@@ -34,11 +35,26 @@ export const QuotedAnnouncementBubble: React.FC<Props> = ({
 }: Props) => {
   const [isExpanded, setisExpanded] = useState(false);
 
-  // If the content is not expanded, limit the message length so that screen readers
-  // don't attempt to read the entire hidden message. This only works for plain-string children
+  /* If the content is not expanded, limit the message length so that screen readers
+    don't attempt to read the entire hidden message. This only works for plain-string children */
   let content = children;
   if (!isExpanded && typeof content === "string") {
-    content = content.substring(0, 300);
+    const previewLength = 100;
+    const firstLineBreak = content.indexOf("\n");
+    /* If we have a line break and the first line of text is shorter than the preview length
+     and another line in the body of text, we should preserve the line break in the preview 
+     and adjust the preview width accordingly */
+    if (firstLineBreak > -1 && firstLineBreak < previewLength - 1) {
+      const allLines = content.split("\n");
+      const firstLine = allLines[0];
+      const longestLine = allLines.reduce((longest, current) =>
+        longest.length > current.length ? longest : current,
+      );
+
+      content = _.padEnd(firstLine, longestLine.length, "\xa0");
```

Comment:
v nice. i appreciate your attention to detail with this fix

Reply:
Actually, I think you will also need to take into account the "Message Details". If that is longer than the message body, we will still see the jumping issue